VERSIONING.md

```diff
@@ -33,3 +33,35 @@ Compose version | Kotlin version
 1.0.0 | 1.5.31
 1.1.1 | 1.6.20
 1.2.0 | 1.7.10 - 1.7.20 (Desktop, Android), 1.7.10 (JS)
+
+### Relation to Jetpack Compose and Compose Multiplatform release cycle
+
+Compose Multiplatform shares a lot of code with [Jetpack Compose](https://developer.android.com/jetpack/compose) for Android developed by Google.
+We keep our release cycles aligned, so the common part is properly tested and stabilized.
+
+When the new minor version of Jetpack Compose is released, we pick the release commit, use it as a base for [Compose Multiplatform fork](https://github.com/JetBrains/androidx), finish new platform features, stabilize all platforms, and release Compose Multiplatform after some time.
```

Comment:
Let's avoid the term "fork", as it can sometimes be interpreted as implying that it is creating a competing/hostile product.  Instead of "Compose Multiplatform fork", perhaps consider "Compose Multiplatform build" or even just "Compose Multiplatform"

Reply:
Or... "use it as a base for the next Compose Multiplatform release" or "use it as a base for the next Compose Multiplatform version"

Reply:
"use it as a base for the next Compose Multiplatform version" looks better, thanks!